spec/factories/users.rb

```diff
@@ -209,6 +217,7 @@
           :with_pii,
           idv_level: :unsupervised_with_selfie,
           user: user,
+          proofing_components: { document_check: 'mock', document_type: 'biometric' },
```

Comment:
To my previous comment, these aren't really accurate to the proofing components of a biometric proofed profile in the real-world. Maybe we can just revert the `proofing_components` changes in this file